force-app/main/default/classes/DonationHistoryController_TEST.cls

```diff
@@ -92,7 +93,17 @@ public class DonationHistoryController_TEST {
         List<Opportunity> recordsOpp = new List<Opportunity> {opportunityHardCreditOfContact0, opportunityAnotherHardCreditOfContact0, opportunityForContact1, anotherOpportunityForContact1};
         // recordsOpp.add(opportunityHardCredit);
         // recordsOpp.add(opportunitySoftCredit);
+
         insert recordsOpp;
+        
+                if (!Schema.sObjectType.Profile.isAccessible()) {
```

Comment:
Indentation is wrong

Reply:
I don't think we need to check for access to Profiles as part of test data setup. The running user for a unit test should be a sysadmin usually, who would have access to profiles. If we need to test other levels of access like a standard user, we can mock those permissions in the test.